doc/installation_instructions/platform/frontier.rst

```diff
@@ -9,6 +9,7 @@ Frontier, however we do have some known limitations:
 
 * For now, only Torch and ONNX runtime models are supported. If you need
   Tensorflow support please contact us
```

Comment:
Not strictly related to this PR, but should this read "For now only Torch models are supported ..." since we do not have a ROCm 6.1 build for ORT?